docs/grid_model/network_subnetwork.md

```diff
@@ -819,9 +820,38 @@ A busbar section is a non impedant element used in a node/breaker substation top
 (internal-connection)=
 ## Internal connection
 
-**Internal connection**  
+[![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/VoltageLevel.NodeBreakerView.InternalConnection.html)<br>
 An internal connection is a non-impedant connection between two components in a voltage level.
```

Comment:
I don' like "components" because it is confusing with connected/synchonous components. We prefer "elements" or devices.